Sources/BSWInterfaceKit/SwiftUI/ViewModifiers/HUDView.swift

```diff
@@ -176,20 +249,28 @@ struct MacHUDModifier: ViewModifier {
         }
     }
 }
-
 #endif
 
-
 struct HUDView: View {
 
+    init(state: HUDState) {
+        self.state = state
+    }
+
     let state: HUDState
 
+    #if canImport(Darwin)
     @ScaledMetric
     private var hudImageSize = 60.0
-    
     @ScaledMetric
     private var hudContentSize = 120.0
-
+    private let backgroundColor = Material.regularMaterial
+    #else
+    private var hudImageSize = 60.0
+    private var hudContentSize = 120.0
+    private let backgroundColor = Color.gray
+    #endif
```

Comment:
Since `struct HUDView` is only available in Darwin platforms, please remove this conditions